### ISSUE TITLE
[HELP] Is it possible to change the output file name for command "submission"??

### ISSUE THREAD

**Issue by yanzhang0219** [skygragon/leetcode-cli#125]
When I run the command: leetcode submission 1, the generated output file name is two-sum.73790064.ac.cpp. I am wondering whether I can change the name such as two-sum.cpp (I just want to remove those series of numbers..)?

Thank you very much.

**Comment by sfdye** [comment]
Don't think there is. At least for me I had to manually rename each file or you can simply write a script to do that.

**Comment by skygragon** [comment]
Yep, recently I'm considering to move those meta into code file, instead of messing filename as it is.

**Comment by sfdye** [comment]
Like to see that happen, e.g. python3 submission with py extension

**Comment by yanzhang0219** [comment]
Cool. Seems @skygragon is coming back to take care of this amazing but drowsy repo. Lol. You are away for a long time and we miss you. Really like leetcode-cli and really appreciate creating such a great tool. I hope this tool can be further polished.
